pkg/api/types.go

```diff
@@ -862,6 +871,9 @@ type Node struct {
 
 	// Status describes the current status of a Node
 	Status NodeStatus `json:"status,omitempty"`
+
+	// Info are information collected on the Node
+	Info NodeInfo `json:"info,omitempty"`
```

Comment:
The internal representation should follow v1beta3 conventions. Therefore, NodeInfo cannot go here. It must go into NodeStatus.